code/game/objects/items/implants/implant.dm

```diff
@@ -87,6 +89,8 @@
 /obj/item/implant/proc/unimplant()
 	SHOULD_CALL_PARENT(TRUE)
 	forceMove(get_turf(implant_owner))
+	if(!implanted)
```

Comment:
why is this after the forcemove?

Reply:
Because if I put it before the forcemove it runtimes.